tests/test_protocol_v7.rs

```diff
@@ -1525,7 +1535,7 @@ mod test_contexts {
             contexts: {
                 let mut m = v7::Map::new();
                 m.insert(
-                    "othervm".into(),
```

Comment:
why did you make this change?

Reply:
I missed this in the previous PR. Intended this method to test a "default" runtime context, and had one for renaming contexts in `test_renamed_contexts`.